.github/scripts/hermetic_library_generation.sh

```diff
@@ -90,9 +90,6 @@ fi
 git show "${target_branch}":"${generation_config}" > "${baseline_generation_config}"
 config_diff=$(diff "${generation_config}" "${baseline_generation_config}" || true)
 
-generator_version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout -pl gapic-generator-java)
```

Comment:
there is a `docker build` call right after this deleted section. That image will contain the SNAPSHOT jar (and the corresponding env var)